run.sh

```diff
@@ -26,21 +26,28 @@ function usage_build()
 {
     echo "Options for subcommand 'build':"
     echo "   -h|--help         print the help info"
-    echo "   -d|--debug        build in debug mode, default is release"
+    echo "   -t|--type         build type: debug|release, default is debug"
```

Comment:
I am not very clear about them, maybe we can add them when needed in the future.